tests/codebase/test_js_license_set.py

```diff
@@ -43,7 +43,7 @@ def test_js_license_set() -> None:
     '''
     os.chdir('bokehjs')
     proc = Popen([
-        "npx", "license-checker", "--production", "--summary", "--onlyAllow", "%r" % ";".join(LICENSES)
+        "npx", "license-checker", "--production", "--summary", "--onlyAllow", "%s" % ";".join(LICENSES)
```

Comment:
This was addressed recently on `branch-2.3`. Is this new?

Reply:
no it was failing for me locally I probably just need to update something, I reverted it

Reply:
You need npm 7.4+. I forgot to update min required version when fixing (again) this issue.